### PR TITLE
Move/refactor boosters into openshift testsuite

### PR DESCRIPTION
**Pull request by juangon**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+                            <includes>
+                                <include>thorntail-v2</include>
+                            </includes>
+                            <excludes>
+                                <exclude>webapp</exclude>
+                            </excludes>
+                        </generator>
+                        <enricher>
+                            <config>
+                                <wildfly-swarm-health-check>
```
Comment (Ladicek) [inline pom.xml]: `<thorntail-v2-health-check>`

**Comment by Ladicek** [comment]
Other than that, I don't see anything obviously wrong, just poor code style (tab indentation??) and consistency issues.

**Comment by juangon** [comment]
Please can you describe those consistency issues?

**Comment by Ladicek** [comment]
I was probably thinking about consistency with the standalone test suite -- names of the test suite modules, names of the `version.*` properties (I think I've seen something like `fabric8-maven-plugin.version` or so) etc.

**Comment by juangon** [comment]
@Ladicek I think I fixed the mentioned issues.

**Comment by Ladicek** [comment]
I actually fixed them locally as well, yesterday, but didn't have time to finish. I hope to do so today.

**Pull request merged**